src/native_background.ts

```diff
@@ -282,6 +282,23 @@ export async function getenv(variable: string) {
     return (await sendNativeMsg("env", { var: variable })).content
 }
 
+export async function clipboard(action: "set" | "get", str: string): Promise<MessageResp> {
```

Comment:
Needs documentation + probably a name that won't conflict with excmds.ts:clipboard()